advanced/Templates/pihole-FTL.service

```diff
@@ -37,8 +37,9 @@ start() {
     chown pihole:pihole /etc/pihole /etc/pihole/dhcp.leases 2> /dev/null
     chown pihole:pihole /var/log/pihole-FTL.log /var/log/pihole.log
     chmod 0644 /var/log/pihole-FTL.log /run/pihole-FTL.pid /run/pihole-FTL.port /var/log/pihole.log
+    chmod 0644 /etc/pihole/macvendor.db
```

Comment:
Is there a specific reason to change the owner/permission of this file? It is read-only for FTL and only updated on FTL updates (via the installer which is `root`)?

Reply:
We're already doing it, I just moved it from the `FTLInstall` function into the service file (and added the leading `0` to match the `chmod` above)

https://github.com/pi-hole/pi-hole/blob/master/automated%20install/basic-install.sh#L2325

The original intention was to move the call to `FTLdetect` higher in the install process, but that caused some different issues. However, in order to move it higher, we needed to get rid of it's usage of the `pihole` user.